Serilog.sln

```diff
@@ -1,25 +1,12 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
```

Comment:
Yikes - SLN file merge hell :*( ... any way we can add the project using VS so the solution file remains a bit less churny? I'm mostly concerned that a big SLN file chance will make the other PRs coming in harder to merge successfully.